package.json

```diff
@@ -65,7 +65,7 @@
     "cozy-harvest-lib": "7.2.3",
     "cozy-intent": "^1.17.1",
     "cozy-realtime": "4.0.5",
-    "cozy-scripts": "6.1.1",
+    "cozy-scripts": "^6.3.1",
```

Comment:
This is not a chore commit. cozy-scripts can inject / modify bundled code.

Also did you test the public view? Is it still working?